package.json

```diff
@@ -34,62 +34,50 @@
   "exports": {
     ".": {
       "types": "./dist/date-picker/app-date-picker.d.ts",
-      "import": "./dist/date-picker/app-date-picker.js",
       "default": "./dist/date-picker/app-date-picker.js"
     },
     "./app-date-picker": {
       "types": "./dist/date-picker/app-date-picker.d.ts",
-      "import": "./dist/date-picker/app-date-picker.js",
       "default": "./dist/date-picker/app-date-picker.js"
     },
     "./app-date-picker-dialog": {
       "types": "./dist/date-picker-dialog/app-date-picker-dialog.d.ts",
-      "import": "./dist/date-picker-dialog/app-date-picker-dialog.js",
       "default": "./dist/date-picker-dialog/app-date-picker-dialog.js"
     },
     "./app-date-picker-input": {
       "types": "./dist/date-picker-input/app-date-picker-input.d.ts",
-      "import": "./dist/date-picker-input/app-date-picker-input.js",
       "default": "./dist/date-picker-input/app-date-picker-input.js"
     },
     "./app-month-calendar": {
       "types": "./dist/month-calendar/app-month-calendar.d.ts",
-      "import": "./dist/month-calendar/app-month-calendar.js",
       "default": "./dist/month-calendar/app-month-calendar.js"
     },
     "./app-year-grid": {
       "types": "./dist/year-grid/app-year-grid.d.ts",
-      "import": "./dist/year-grid/app-year-grid.js",
       "default": "./dist/year-grid/app-year-grid.js"
     },
     "./date-picker": {
       "types": "./dist/date-picker/date-picker.d.ts",
-      "import": "./dist/date-picker/date-picker.js",
       "default": "./dist/date-picker/date-picker.js"
     },
     "./date-picker-dialog": {
       "types": "./dist/date-picker-dialog/date-picker-dialog.d.ts",
-      "import": "./dist/date-picker-dialog/date-picker-dialog.js",
       "default": "./dist/date-picker-dialog/date-picker-dialog.js"
     },
     "./date-picker-input": {
       "types": "./dist/date-picker-input/date-picker-input.d.ts",
-      "import": "./dist/date-picker-input/date-picker-input.js",
       "default": "./dist/date-picker-input/date-picker-input.js"
     },
     "./month-calendar": {
       "types": "./dist/month-calendar/month-calendar.d.ts",
-      "import": "./dist/month-calendar/month-calendar.js",
       "default": "./dist/month-calendar/month-calendar.js"
     },
     "./root-element": {
       "types": "./dist/root-element/root-element.d.ts",
-      "import": "./dist/root-element/root-element.js",
       "default": "./dist/root-element/root-element.js"
     },
     "./year-grid": {
       "types": "./dist/year-grid/year-grid.d.ts",
-      "import": "./dist/year-grid/year-grid.js",
```

Comment:
IIUC, `import` is redundant as this package only supports ESM and `default` can do what we want it to do but I chose to keep it back then. I gues it's safe to remove those. Thanks for the removal.